release_notes/v1.0.0.md

```diff
@@ -0,0 +1,28 @@
+# Web SDK is now 1.0!
+
+We cooked up a big release for this milestone. We already had cool features like the actions buttons on our mobile SDKs, but now, you can also have them! We also packed this release with a lot of fixes and new things that will surely be useful to a lot of you guys.
+
+# Web SDK is seeing some action!
+![SupportKit Web Action Buttons](http://docs.supportkit.io/images/action_button_web.png)
+
+The Web SDK now features Actions!
+You can now send Actions along with messages to your users. This makes it easier to direct your users to some part of your website and help them find exactly what they need.
+
+See our [docs](http://docs.supportkit.io/javascript#sending-action-buttons) for all the details.
+
+# Login, Logout, JWT and your conversation everywhere
+
+You can now specify a `userId` and (optionally) a JSON Web Token (JWT) to securely log users in and out of your application. Every user session will hold its own conversation and properties. That same user is now able to log in into their iOS device, Android device, or on your website and retain their conversation!
+
+As usual, see our [docs](http://docs.supportkit.io/javascript#users-on-multiple-devices) for the details.
+
+# Fixes and improvements
+
+- We rewrote the whole communication layer to not rely on `jQuery.ajax`. This means no more problems with Authorization headers when you are using the JWT in your own app.
+- You can now send links in your messages. We added a link detection feature that will render them correctly. It's currently pretty basic and will detect anything under the following format: `procotol://uri` so you could send a normal url or even some link to iTunes or the Play store if you wanted to. We are not detecting emails for now, but you could achieve that with action buttons!
+- We worked hard on building a better experience on mobile. The widget should now use the whole screen when opened.
+- A lot of CSS fixes
+
+# API Additions
+
+1. SupportKit global now has `login` and `logout`. Check out our [README](https://github.com/supportkit/supportkit-js) for a more complete API description.
```

Comment:
no additions in `init` the jwt is passed in the options and it was already in the doc for some reason.